packages/textlint/src/index.ts

```diff
@@ -26,3 +26,6 @@ export { TextFixEngine } from "./textfix-engine";
 
 // Core API for linting a **single** text or file.
 export { TextLintCore } from "./textlint-core";
+
+// Config API for loading textlint config
+export { Config } from "./config/config";
```

Comment:
Config is unstable API. We not want to export it.
(It will be changed in the future)

Reply:
Well, it's already a public API by `import { Config } from 'textlint/lib/config/config';`

But after upgrading to v12, there are `textlint/lib/src/config/config` for cjs vs `textlint/module/src/config/config` for esm.

It's just bad for a consumer.

Maybe `exports` field in `package.json` should be used.

Reply:
Yes. It is internal API.